src/content/docs/codestream/start-here/codestream-new-relic.mdx

```diff
@@ -0,0 +1,228 @@
+
+Once you have your API key, in **Observability** click **Connect to New Relic One**, then paste your API key and click **Connect**.
+
+<Callout variant="caution">
+New Relic users can share stacktrace errors on CodeStream. Once you've connected CodeStream to New Relic, any new users you add to your CodeStream organization will be able to see those errors.
+</Callout>
+
+## See your errors and what's causing them [#errors]
+
+Once you've connected CodeStream and New Relic, use workloads and errors inbox to jump to the offending code in your IDE.
+
+Requirements for opening stack trace errors in your IDE:
+* [New Relic account](https://newrelic.com/signup) (If you don't have a New Relic account, sign up at [newrelic.com/signup](https://newrelic.com/signup). It's free, forever!)
+* [New Relic user key](https://one.newrelic.com/launcher/api-keys-ui.api-keys-launcher)
+* [CodeStream and New Relic connection](#connect)
+* [Data being reported to New Relic via APM monitoring](https://one.newrelic.com/launcher/nr1-core.explorer?pane=eyJuZXJkbGV0SWQiOiJucjEtY29yZS5saXN0aW5nIn0=&cards[0]=eyJuZXJkbGV0SWQiOiJucjEtaW5zdGFsbC1uZXdyZWxpYy5ucjEtaW5zdGFsbC1uZXdyZWxpYyIsImFjdGl2ZUNvbXBvbmVudCI6IlZUU09FbnZpcm9ubWVudCIsInBhdGgiOiJndWlkZWQifQ==)
+* [A workload for errors inbox](/docs/new-relic-one/use-new-relic-one/workloads/use-workloads/#create)
+* [An error](/docs/apm/apm-ui-pages/errors-inbox/errors-inbox/#error-details)
+* [For APM errors, your repository's commit hash and release tag](#apm)
+
+New Relic errors inbox is a single place to proactively detect, prioritize, and take action on your errors before they impact customers. With CodeStream, you can jump from an error directly to the offending code in your IDE.
+
+<Callout variant="tip">
+    Limited to APM errors.
+</Callout>
+
+From **[one.newrelic.com/](https://one.newrelic.com/)**, go to **Errors Inbox**, click a stack trace error, then click **Open in IDE**.
+
+## APM errors and CodeStream [#apm]
+
+To use CodeStream's **Open in IDE** with your APM stack trace errors, configure your APM agent with your application's [commit sha](https://git-scm.com/book/en/v2/Git-Tools-Revision-Selection)
+and/or your [release tag](https://git-scm.com/book/en/v2/Git-Basics-Tagging) associated with the running version of your software.  
+
+CodeStream only needs the first seven characters of your commit sha (for example, `734713b`) to make this connection, but you can include the entire sha.  
+
+Alternately, you can use a release tag (such as `v0.1.209` or `release-209`) for CodeStream to find the correct version of your code.
+
+For New Relic APM, the commit and/or release tag (`tags.commit` and `tags.releaseTag`) are added as attributes on `Transaction` and `TransactionError` events. The APM agent protocol provides an easy way to set this up via environment variables.  
```

Comment:
If you're not going to describe how to do this, can you link to instructions? I think "easy" is a word you might use with great care :)

Reply:
That "easy" slipped through :D

Reply:
Super valid point that this needs to be explained.

Reply:
I've added links to the config docs for each language